README.md

```diff
@@ -54,7 +54,8 @@ Options:
                                             group will be generated in AWS
   -n, --allow-any-temporary-security-group  The temporary security group will allow SSH connections 
                                             from any IP address (0.0.0.0/0), otherwise allow the subnet's block
-  -p, --aws-public-ip                       Launch instances with a public IP
+  -p, --aws-public-ip                       Launch instances with a public IP and use that IP for SSH
+  -q, --associate-public-ip                 Launch instances with a public IP but don't use that IP for SSH
```

Comment:
I think the naming is throwing me off a bit. Given the difference here is whether it uses private or public ip to ssh could it be something like `associate-private-ip`?

Reply:
That is confusing to me because an EC2 instance always has a private IP.